Formula/pinot.rb

```diff
@@ -28,6 +28,19 @@ def install
   end
 
   test do
-    assert_match("StartController", shell_output("#{bin}/pinot-admin -h 2>&1"))
+    fork do
+      system opt_bin/"pinot-admin", "StartZookeeper", "-zkPort", "7689"
+    end
+
+    sleep 10
+
+    fork do
+      system opt_bin/"pinot-admin StartController -zkAddress localhost:7689 -controllerPort 6969"
+    end
+
+    # wait for startup
+    sleep 40
+
+    assert_match("HTTP/1.1 200 OK", shell_output("curl -i http://localhost:6969 2>&1"))
```

Comment:
```suggestion
    zkport = free_port
    controller_port = free_port
    fork do
      exec opt_bin/"pinot-admin", "StartZookeeper", "-zkPort", zkport
    end

    sleep 10

    fork do
      exec opt_bin/"pinot-admin", "StartController", "-zkAddress", "localhost:#{zkport}", "-controllerPort", controller_port
    end

    # wait for startup
    sleep 40

    assert_match("HTTP/1.1 200 OK", shell_output("curl -i http://localhost:#{controller_port} 2>&1"))
```
Do we need to kill the server after the test?

Reply:
Yes, that would be a good idea. How is that done via Ruby? Do you get the PID and then `kill -9`?

Reply:
Something like
```ruby
test do
  zkpid = fork do
    exec ...
  end
  ...
  controller_pid = fork do
    exec ...
  end
  ...
ensure
  Process.kill "TERM", zkpid
  Process.kill "TERM", controller_pid
  Process.wait zkpid
  Process.wait controller_pid
end
```
should work.

Reply:
Done and tested. Thanks again! I'm starting to like Ruby. 💯